Sources/Sword/Gateway/GatewayHandler.swift

```diff
@@ -53,15 +56,40 @@ extension GatewayHandler {
       return
     }
     
-    let config = WebSocketClient.Configuration(maxFrameSize: 1 << 31)
+    var config = WebSocketClient.Configuration(maxFrameSize: 1 << 31)
+    
+    func _isSSL() -> Bool {
```

Comment:
Discord is always going to routed through `wss` and port 443 (if /gateway/bot didnt already give us one). You really only need to do:
```swift
let config = WebSocketClient.Configuration(
  tlsConfiguration: .forClient(),
  maxFrameSize: 1 << 31
)
```

Reply:
OK. I rewrote it at 401a68a.